pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -45,6 +45,8 @@ type staticPolicy struct {
 	systemReserved systemReservedMemory
 	// topology manager reference to get container Topology affinity
 	affinity topologymanager.Store
+	// init container's memory and hugepages reservation that can be reused by app containers in the pod
+	memoryToReuse map[string]map[v1.ResourceName]state.Block
```

Comment:
It could be also a `map[string][]state.Block` to not duplicate `Block.Type` value but making it a nested map makes logic in few places simpler (without nested loops).